src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Engine/TriviaData.cs

```diff
@@ -18,16 +18,12 @@ internal abstract class TriviaData
 {
     protected const int TokenPairIndexNotNeeded = int.MinValue;
 
-    private readonly string _language;
-
-    protected TriviaData(SyntaxFormattingOptions options, string language)
+    protected TriviaData(LineFormattingOptions options)
     {
         Options = options;
-        _language = language;
     }
 
-    protected SyntaxFormattingOptions Options { get; }
-    protected string Language => _language;
+    protected LineFormattingOptions Options { get; }
```

Comment:
all consumers only need the limited amount of info exposed by LineFormattingOPtions,  not the full info in SyntaxFormattingOptions.